convertToSingleHost.js

```diff
@@ -5,147 +5,161 @@ const hosts = ["excel", "outlook", "powerpoint", "word"];
-        // copy over host-specific taskpane test code to test-taskpane.ts
-        const testTaskpaneContent = await readFileAsync(`./test/src/${host}-test-taskpane.ts`, "utf8");
-        const updatedTestTaskpaneContent = testTaskpaneContent.replace(`../../src/taskpane/${host}`, `../../src/taskpane/taskpane`);
-        await writeFileAsync(`./test/src/test-taskpane.ts`, updatedTestTaskpaneContent);
-
-        // update ui-test.ts to only run against specified host
-        const testContent = await readFileAsync(`./test/ui-test.ts`, "utf8");
-        const updatedTestContent = testContent.replace(`const hosts = ["Excel", "Word"]`, `const hosts = ["${host}"]`);
-        await writeFileAsync(`./test/ui-test.ts`, updatedTestContent);
-
-        // delete all host-specific test files after converting to single host
-        hosts.forEach(async function (host) {
-            if (host == "excel" || host == "word") {
-                await unlinkFileAsync(`./test/src/${host}-test-taskpane.ts`);
-            }
-        });
-    }
-    else {
-        deleteFolder(path.resolve(`./test`));
-    }
-
-    // delete all host-specific files
+  // copy host-specific manifest over manifest.xml
+  const manifestContent = await readFileAsync(`./manifest.${host}.xml`, "utf8");
+  await writeFileAsync(`./manifest.xml`, manifestContent);
+
+  // copy over host-specific taskpane code to taskpane.ts
+  const srcContent = await readFileAsync(`./src/taskpane/${host}.ts`, "utf8");
+  await writeFileAsync(`./src/taskpane/taskpane.ts`, srcContent);
+
+  // delete all test files by default for now - eventually we want to convert the tests by default
+  if (convertTest && (host === "excel" || host === "word")) {
+    // copy over host-specific taskpane test code to test-taskpane.ts
+    const testTaskpaneContent = await readFileAsync(`./test/src/${host}-test-taskpane.ts`, "utf8");
+    const updatedTestTaskpaneContent = testTaskpaneContent.replace(`../../src/taskpane/${host}`, `../../src/taskpane/taskpane`);
+    await writeFileAsync(`./test/src/test-taskpane.ts`, updatedTestTaskpaneContent);
+
+    // update ui-test.ts to only run against specified host
+    const testContent = await readFileAsync(`./test/ui-test.ts`, "utf8");
+    const updatedTestContent = testContent.replace(`const hosts = ["Excel", "Word"]`, `const hosts = ["${host}"]`);
+    await writeFileAsync(`./test/ui-test.ts`, updatedTestContent);
+
+    // delete all host-specific test files after converting to single host
     hosts.forEach(async function (host) {
-        await unlinkFileAsync(`./manifest.${host}.xml`);
-        await unlinkFileAsync(`./src/taskpane/${host}.ts`);
+      if (host == "excel" || host == "word") {
+        await unlinkFileAsync(`./test/src/${host}-test-taskpane.ts`);
+      }
     });
-
-    // delete the .github folder
-    deleteFolder(path.resolve(`./.github`));
-
-    // delete this script
-    await unlinkFileAsync("./convertToSingleHost.js");
+  }
```

Comment:
Did you run prettier?  The } should be on the same line as `else {` here.

Reply:
Yes I have (prettier changes the CRLF to LF) and it didn't change this.  It's been that way for over a year.  My change just updated the tab spacing to make merging simpler (since it matches the taskpane project . . . which also has these on separate lines).